packages/client/src/game-core/PlayerStates/commonUtils.ts

```diff
@@ -0,0 +1,20 @@
+import { Player } from '../Player'
+import { State } from '../State'
+import { InputHandler } from '../input/InputHandler'
+
+export type PlayerState = State<Player>
+
+export function pressJump(input: InputHandler) {
```

Comment:
Функция кажется возвращает булево значение, но по названию выглядит будто выполняет действие прыжка, а не проверяет его

```suggestion
export function pressedJump(input: InputHandler) {
```

Reply:
Можно даже лучше: `isJumpPressed`